### DISCUSSION TITLE
Login page doesn't load remotely on iPhone

### DISCUSSION THREAD

**Discussion by gokou340** [nextcloud/all-in-one#2650] [Questions] [answered]
I just setup Nextcloud-AIO via Portainer with the docker-compose file. It sits behind Nginx-Proxy-Manager and my external DNS is Cloudflare with proxy turned off. When I am on mobile data, and browse to my nextcloud URL, the login page doesn't load, sometimes it takes 2+ minutes for the background to load, but the login boxes do not load. If I connect to my wifi, and reload the page, it loads relatively quickly. At first I thought it was an issue with remote access, but when I use a laptop or PC on an external network, it loads fine. I've tried multiple iphones. I have not tried an android, as I don't have one to test with.

If I download the iOS Nextcloud app, when I put in my https:// URL, it tries to load, but never loads the login prompt and sits spinning. When I connect to my wifi network, it loads instantly.

Things I have done to troubleshoot:
- Spin up a brand new VM and run Ubuntu 20.04 with docker and deploy Nextcloud, same result.
- Bypass Nginx Proxy manager and have the connection go from external directly to the internal server on port 80 and port 443, same issue.

I'm not sure where to go from here and what else I should try. Any tips or suggestions would be appreciated!

**Comment by szaimen** [comment]
Hi, which AIO version are you running?

**Reply by gokou340** [reply]
Looks like it is on 26.0.2

**Reply by szaimen** [reply]
No, I was asking for the AIO version in the AIO interface.

**Reply by gokou340** [reply]
Oh sorry. Looks like it is 5.2.2

**Comment by szaimen** [comment]
Can you follow https://github.com/nextcloud/all-in-one/discussions/2475?

**Reply by gokou340** [reply]
I tried this, but nothing changed. When I would reload the webpage, it still wouldn't load the login prompt, and there were no logs in the admin logging.

**Reply by szaimen** [reply]
Hm... Does it load remotely on a different non-ios device with firefox or chrome?

**Reply by gokou340** [reply]
On a normal remote computer using Edge, it loads without issue.

I don't have an android mobile device to test with unfortunately

**Reply by gokou340** [reply]
Is there any logs I can look at on the container to see what may be happening? As of now, Nextcloud is useless if I can't access it remotely.

**Comment by szaimen** [comment, marked as answer]
> On a normal remote computer using Edge, it loads without issue.

This sounds like a specific compatibility issue with safari then. Please report this to https://github.com/nextcloud/server/issues

**Reply by gokou340** [reply]
What information would you like me to put in the issues area? Since it only happens when I'm remote and not on the local wifi.
